src/frontend/src/views/Project/Files/Console/index.tsx

```diff
@@ -66,6 +79,10 @@ export default class Xterm extends React.Component<ConsoleProps, ConsoleState> {
   componentDidMount() {
     this.term.open(this.container);
     this.setState({ input: true, line: 1, currString: "" });
+    if  (this.props.display.props.appState.currentProject) {
+      let consoleText = this.props.display.props.appState.currentProject.consoleText;
+      if (consoleText) this.term.write(consoleText);
+    }

```

Comment:
I've removed the file prop from DisplayFile -- I haven't checked if this fixes this PR, but please look if you have a chance.